test/integration/targets/vault/symlink/get-password-symlink

```diff
@@ -0,0 +1 @@
+../script/vault-secret.sh
```

Comment:
Git tends to act funny with symlinks in some corner cases. Would you consider creating this in `runme.sh` dynamically + gitignoring it?